.github/workflows/pre-commit.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
+      - uses: actions/setup-python@v4
```

Comment:
This should be reverted.
Already merged here https://github.com/OCA/management-system/pull/454/commits/0e46bd9add9f5c90fa1b92b3146ff1ea501afb1d